jax/numpy/lax_numpy.py

```diff
@@ -3089,6 +3089,18 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
   return true_divide(dot(X, X_T.conj()), f).squeeze()
 
 
+@_wraps(onp.convolve)
+def convolve(x, y, mode='full'):
```

Comment:
I suspect you need some type promotion logic here. For example, does this implementation work similarly to:
```
onp.convolve(onp.arange(5), onp.arange(3, dtype=onp.float32))
```

Most likely what you need is the line:
```
x, y = _promote_types(x, y)
```